srcpkgs/librsvg/template

```diff
@@ -22,13 +24,18 @@ if [ -z "$CROSS_BUILD" ]; then
 	build_options_default="vala gir"
 fi
 
-pre_configure() {
-	autoreconf -fi
+post_configure() {
+	cat <<- EOF > .cargo/config
+[target.${RUST_TARGET}]
```

Comment:
$RUST_TARGET is not defined for native builds

Reply:
Yes, was only meant for testing - thanks for the review though!